Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -40,11 +40,6 @@ Some properties of the operations are also used to discuss general tools on ordi
 * `IsNormal`: a function `f : Ordinal → Ordinal` satisfies `IsNormal` if it is strictly increasing
   and order-continuous, i.e., the image `f o` of a limit ordinal `o` is the sup of `f a` for
   `a < o`.
-* `enumOrd`: enumerates an unbounded set of ordinals by the ordinals themselves.
-* `sup`, `lsub`: the supremum / least strict upper bound of an indexed family of ordinals in
```

Comment:
I'd say we shouldn't remove the docs for things which aren't deprecated, at least until they actually are

Reply:
Fair enough, I'll only remove the stuff about `enumOrd` for now then.